pages/documentation/developer/repositories/index.md

```diff
@@ -28,17 +28,11 @@ This [repository](https://github.com/powsybl/powsybl-balances-adjustment) provid
 **Reviewers:** [phiedw](https://github.com/phiedw), [miovd](https://github.com/miovd)   
 **Committers:** [phiedw](https://github.com/phiedw), [miovd](https://github.com/miovd)  
 
-### [powsybl-single-line-diagram](powsybl-single-line-diagram.md) [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-single-line-diagram.svg?sort=semver)](https://github.com/powsybl/powsybl-single-line-diagram/releases/)
-This [repository](https://github.com/powsybl/powsybl-single-line-diagram) provides modules to generate single line diagrams.
+### [powsybl-diagram](powsybl-diagram.md) [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-diagram.svg?sort=semver)](https://github.com/powsybl/powsybl-diagram/releases/)
+This [repository](https://github.com/powsybl/powsybl-diagram) provides modules to generate single-line diagrams and network-area diagrams.
```

Comment:
As network-area diagram is a "powsybl" name, maybe we should say something like "network graph diagrams"?